cmd/bpf2go/gen/compile.go

```diff
@@ -40,7 +41,8 @@ func Compile(args CompileArgs) error {
 		"-mcpu=v1",
 	}
 
-	cmd := exec.Command(args.CC, append(overrideFlags, args.Flags...)...)
+	ccParts := strings.Fields(args.CC)
```

Comment:
Doing this here means that `CC` is kind of like an implicit additional `Flags` argument, without any documentation. I think it makes more sense to do this in `main()` where we pull CC out of flags. There we can split it and prepend it to CompileArgs.Flags.

Reply:
Sure, I can move the split earlier in the sequence. We'd need to reorder the append here as well then:

```
append(args.Flags, overrideFlags...)
```

Reply:
Done.